src/App.tsx

```diff
@@ -1,7 +1,37 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string | null,
+};
+
+export class App extends Component<{}, State> {
+  state: Readonly<State> = {
+    pressedKey: null,
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.keyPress);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.keyPress);
+  }
+
+  keyPress = (event: KeyboardEvent) => {
+    this.setState({ pressedKey: event.key });
```

Comment:
let's use `event.code` to check if the pressed key is `space` and write `[space]` if so
<img width="200" alt="image" src="https://user-images.githubusercontent.com/55802630/182804945-3088e966-2abb-4a2b-a525-140c2dc25704.png">